src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -39,12 +54,36 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   scheduleStartHour,
   personalizedEvent: isPersonalizedEvent = false,
 }) => {
+  const eventRef: RefObject<HTMLDivElement> = useRef(null);
+  const isMinimallyLongEvent = event.duration_minutes >= MIN_LONG_EVENT_LENGTH;
+  const isShortEvent = event.duration_minutes <= SHORT_EVENT_LENGTH;
+  const isEventLong = event.duration_minutes >= LONG_EVENT_LENGTH;
+
   const { userId } = useUser();
+  const {
+    isShown: isExpanded,
+    hide: hideExpanded,
+    show: showExpanded,
+  } = useShowHide();
+  const {
+    isShown: isExpandedStatic,
+    toggle: toggleExpandedStatic,
+    hide: hideExpandedStatic,
+  } = useShowHide();
+
+  const isBookmarkDisplayed =
+    isMinimallyLongEvent || isExpanded || isExpandedStatic;
+  const isContentDisplayed = isExpanded || isExpandedStatic || isEventLong;
+  const isShowExpand = (!isShortEvent || isExpanded) && !isEventLong;
 
   // @debt ONE_HOUR_IN_MINUTES is deprectated; refactor to use utils/time or date-fns functions
-  const eventWidthPx =
-    (event.duration_minutes * SCHEDULE_HOUR_COLUMN_WIDTH_PX) /
-    ONE_HOUR_IN_MINUTES;
+  const eventWidthPx = getMinuteValue(
+    event.duration_minutes * SCHEDULE_HOUR_COLUMN_WIDTH_PX
+  );
+
+  const expandedEventPx = getMinuteValue(
+    LONG_EVENT_LENGTH * SCHEDULE_HOUR_COLUMN_WIDTH_PX
+  );
```

Comment:
Proper fix for above comment:
```suggestion
  const eventWidthPx = minutesToHours(event.duration_minutes) * SCHEDULE_HOUR_COLUMN_WIDTH_PX;

  const expandedEventPx = minutesToHours(LONG_EVENT_LENGTH) * SCHEDULE_HOUR_COLUMN_WIDTH_PX;
```

Reply:
will do once we resolve https://github.com/sparkletown/sparkle/pull/1733#discussion_r665961085

Reply:
changed `expandedEventPx` and `eventWidthPx` to use datefns in  a265519
but `eventWidthPx` going to look this way `minutesToHours(event.duration_minutes * SCHEDULE_HOUR_COLUMN_WIDTH_PX)` since `minutesToHours(event.duration_minutes)` would always return 0 for events shorter than 60 minutes due to the fact that `minutestohours` uses math floor